samples/subsys/ipc/ipm_mcux/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+set(BOARD lpcxpresso54114_m4)
```

Comment:
Add a license header and copyright. Do this for all the new files

Reply:
Fixed